Tools/machines/leonardo-cineca/job.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+#SBATCH --time=02:00:00
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=4
+#SBATCH --ntasks-per-socket=4
+#SBATCH --cpus-per-task=8
+#SBATCH --gpus-per-node=4
+#SBATCH --gpus-per-task=1
+#SBATCH --mem=494000
+#SBATCH --partition=boost_usr_prod
+#SBATCH --job-name=<job name>
+#SBATCH --gres=gpu:4
+#SBATCH --err=job.err
+#SBATCH --out=job.out
+#SBATCH --account=<project id>
+#SBATCH --mail-type=ALL
+#SBATCH --mail-user=<mail>
+
+source ~/leonardo_gpu_warpx.profile
```

Comment:
Do we really need to source this in the job script?

Reply:
Yes, otherwise it gives an error in loading shared libraries.

Reply:
@MartaGalbi can you try to:
- log in
- source the file
- submit the job?

In nearly all modern batch systems, especially slurm, the job inherits/copies your environment as loaded at submission time.

Reply:
You are right. If I source the file before submitting the job, it works.